projects/plugins/jetpack/extensions/extended-blocks/core-embed/descript.js

```diff
@@ -0,0 +1,25 @@
+/**
+ * WordPress dependencies
+ */
+import { registerBlockVariation } from '@wordpress/blocks';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { DescriptIcon } from '../../shared/icons';
+/*
+ * New `core/embed` block variation.
+ */
+
```

Comment:
Maybe it would be better like this?

```suggestion

/*
 * New `core/embed` block variation.
 */
```